### ISSUE TITLE
export 'default' (imported as 'InlineSvg') was not found in 'vue-inline-svg' 

### ISSUE THREAD

**Issue by imransilvake** [shrpne/vue-inline-svg#52]
Since yesterday I am getting an error of import. Looks like coming from `3.1.1`. Is something changed?

```
export 'default' (imported as 'InlineSvg') was not found in 'vue-inline-svg' (module has no exports)
```

<img width="752" alt="Screenshot 2023-02-05 at 10 25 00" src="https://user-images.githubusercontent.com/13087225/216811314-9bedda44-da3d-4037-b635-97e651795ad6.png">

**Comment by shrpne** [comment]
Well, the one thing that changed, is `"type": "module"` field in package.json, but i think it should not affect importing vue-inline-svg

I can't reproduce the bug in vite3 and vite4

example with vite4 https://stackblitz.com/edit/vitejs-vite-osyaxy?file=src/App.vue

**Comment by imransilvake** [comment]
could it be that the bug exist in vue-cli with vue-cli? because I am using vue-cli

**Comment by shrpne** [comment]
what version of vue-cli?

**Comment by imransilvake** [comment]
Version:
<img width="321" alt="Screenshot 2023-02-06 at 09 20 16" src="https://user-images.githubusercontent.com/13087225/216920290-34835b5b-e820-4e9f-8cc1-d048affd4d2b.png">

**Comment by ejerskov** [comment]
I have the same issue

**Comment by 9mm** [comment]
Same issue

```
  "devDependencies": {
    "@babel/core": "^7.12.16",
    "@babel/eslint-parser": "^7.12.16",
    "@vue/cli-plugin-babel": "~5.0.0",
    "@vue/cli-plugin-eslint": "~5.0.0",
    "@vue/cli-plugin-router": "~5.0.0",
    "@vue/cli-service": "~5.0.0",
    "@vue/eslint-config-airbnb": "^6.0.0",
    "autoprefixer": "^10.4.2",
    "eslint": "^8.18.0",
    "eslint-plugin-import": "^2.25.3",
    "eslint-plugin-vue": "^9.1.0",
    "eslint-plugin-vuejs-accessibility": "^2.0.0",
    "postcss": "^8.4.6",
    "tailwindcss": "^2.2.19"
  }
```

**Comment by shrpne** [comment]
Should be fixed in v3.1.2
